doc_template/examples_root/examples/nb/ecephys_receptive_fields.ipynb

```diff
@@ -53,11 +53,25 @@
    "metadata": {},
```

Comment:
<p>Change to <code>rf_mapping.get_receptive_field</code>  and remove <strong>NOTE</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/AllenSDK/pull/1603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>